dbcon/joblist/distributedenginecomm.cpp

```diff
@@ -417,14 +430,14 @@ void DistributedEngineComm::Listen(boost::shared_ptr<MessageQueueClient> client,
   for (map_tok = fSessionMessages.begin(); map_tok != fSessionMessages.end(); ++map_tok)
   {
     map_tok->second->queue.clear();
-    ++map_tok->second->unackedWork[0];
+    (void)atomicops::atomicInc(&map_tok->second->unackedWork[0]);
     map_tok->second->queue.push(sbs);
   }
   lk.unlock();
 
   if (fIsExeMgr)
   {
-    const uint32_t originalPMCount = pmCount;
+    decltype(pmCount) originalPMCount = pmCount;
```

Comment:
Why not just `const auto`?

Reply:
Originally that was written before we have C++17/20.